File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.java
class Solution {
    public int maximumCount(int[] nums) {
        int pos = 0;
        int neg = 0;

        for(int i : nums){
            if(i > 0)
                pos++;
            
            if(i < 0)
                neg++;
        }
        return Math.max(pos, neg);
    }
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.cs
public class Solution {
    public int MaximumCount(int[] nums) {
        int p = 0, n = 0;
        foreach(int i in nums){
            if(i>0)
                p++;
            else if(i<0)
                n++;
        }
        
        return Math.Max(p,n);
    }
}